requirements.txt

```diff
@@ -1,2 +1,4 @@
 appdirs==1.4.4
 ttkthemes==3.2.2
+playsound==1.2.2
+PyObjC==8.1; sys_platform == "darwin"
```

Comment:
Why is this needed? I think it deserves a short comment explaining that it has to do with `playsound`.